### DISCUSSION TITLE
重复：快进 快退 关键帧

### DISCUSSION THREAD

**Discussion by jiang-yuan** [hooke007/mpv_PlayKit#159] [回收]
![image](https://user-images.githubusercontent.com/36291537/182537470-09771fdf-8b3d-48fe-9fa5-cf76b819df9f.png)
这是默认设置,但是我每次按方向键进度条都动10秒,按一次好像触发了按两次方向键的效果?

**Comment by hooke007** [comment]
https://github.com/hooke007/MPV_lazy/discussions/13
https://github.com/hooke007/MPV_lazy/discussions/76

**Reply by jiang-yuan** [reply]
我感觉我的和他们有点不一样啊,他们好像随机的,我的每次都是两个5秒

**Reply by hooke007** [reply]
五秒内不存在关键帧自然就跳到下一个关键帧。不想用关键帧就用精确跳转

**Comment by jiang-yuan** [comment]
我以前用potplayer的时候记得可以取消勾选跳转关键帧,然后每个视频不管编码格式都跳得准确了
mpv有没有这种选项?
&nbsp;


Jiang&nbsp;Yuan
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年8月3日(星期三) 晚上9:04
收件人: ***@***.***&gt;; 
抄送: "Yuan ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [hooke007/MPV_lazy] 为啥input.conf默认设置按方向键是快进5秒,每次按了都是变成10秒 (Discussion #159)





 
五秒内不存在关键帧自然就跳到下一个关键帧
 
&mdash;

**Reply by hooke007** [reply]
参阅前面的引用
https://github.com/hooke007/MPV_lazy/discussions/13#discussioncomment-1183629
https://github.com/hooke007/MPV_lazy/discussions/76#discussioncomment-1871930 （你主楼发的图上就有示例）
